server/src/main/resources/application.conf

```diff
@@ -62,7 +62,7 @@ akka {
   extensions = []
 
   # Toggles whether threads created by this ActorSystem should be daemons or not
-  daemonic = off
+  daemonic = on
```

Comment:
This is done as the `ActorSystem` is now shared between all `Scassandra` instances, and thus does not shutdown.  Making `daemonic` true allows the JVM to exit.